guides/common/modules/proc_configuring-your-deployment-to-run-ansible-roles.adoc

```diff
@@ -19,13 +19,23 @@ To enable the Ansible plug-in in {Project}, enter the following command:
 
 endif::[]
 
+[id="Ansible-paths_{context}"]
+.Ansible paths
 
-Complete this procedure to configure your {Project} deployment to run Ansible roles.
+{Project} imports Ansible roles and variables from paths based on configuration in `/etc/ansible/ansible.cfg`.
+{Project} then runs imported roles from paths based on configuration in `/etc/foreman-proxy/ansible.cfg`.
+In both cases, {Project} reads the paths from `roles_path` and `collections_path` directives.
```

Comment:
Maybe last comment, we should recommend the user to keep thes two cfg files in sync, otherwise they will import roles that can't be ran or they will not see roles they can run. Otherwise :+1: from me